pygmt/tests/test_grdimage.py

```diff
@@ -146,18 +146,16 @@ def test_grdimage_over_dateline(xrgrid):
 
 
 @check_figures_equal()
-@pytest.mark.parametrize("lon0", [0, 123, 180])
-@pytest.mark.parametrize("proj_type", ["H", "W"])
-def test_grdimage_central_meridians(grid, proj_type, lon0):
+def test_grdimage_central_meridians(grid):
```

Comment:
Err, could you revert this change please. This parametrize test was set up explicitly at https://github.com/GenericMappingTools/pygmt/pull/560#discussion_r484011483 to catch a bug (#390).

Reply:
Will do; I had just assumed it was repeatedly testing the same arguments. On that note, how should this handle a reference image then, since it produces 6 different images?

Reply:
Actually, I would recommend to keep this test on `@check_figures_equal` for now because it's quite a serious bug we want resolved at some point (differences between plotting NetCDF and xarray grids should not exist).

Reply:
Sounds good!

Reply:
Yes, just for now please :smile:

Reply:
Should be fixed (took me too long to realize there would be errors when I merged with GitHub)!